### PR TITLE
Adds in a new way to create disinfectant.

### PR DESCRIPTION
**Pull request by Broken-Admin**
Adds powdered disinfectant. It can be mixed with water to create disinfectant, or used by it's self.

### CODE REVIEW

**Comment by Coolthulhu** [comment]
Antiseptic powder already exists

**Comment by Broken-Admin** [comment]
This isn't antiseptic powder, it's like whats used for beer making.

**Comment by Broken-Admin** [comment]
Should I put it bars?

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -561,6 +561,29 @@
         }
     },
     {
+        "id" : "disinfectant_powdered",
+        "type" : "COMESTIBLE",
+        "comestible_type" : "MED",
+        "name" : "powdered disinfectant",
+        "description" : "A powdered form of disinfectant. Could be used as a disinfectant, or mixed with water to become disinfectant",
```
Comment (Night-Pryanik) [inline data/json/items/comestibles/med.json]: Double space after period in the first sentence. Period at the end of the second sentence. 
Also, maybe rephrase second sentence a bit? Like "Could be used as is, or mixed with water to become more potent version."?

Comment (Broken-Admin) [reply data/json/items/comestibles/med.json]: I did fix this in my personal version, I supposed I forgot to do this in the git files.

**Comment by Broken-Admin** [comment]
How's that?

File: data/json/items/comestibles/med.json [hunk 2]
```diff
@@ -564,13 +564,14 @@
         "id" : "disinfectant_powdered",
         "type" : "COMESTIBLE",
         "comestible_type" : "MED",
-        "name" : "powdered disinfectant",
-        "description" : "A powdered form of disinfectant. Could be used as a disinfectant, or mixed with water to become disinfectant",
+        "name" : "cleaning powder",
+        "name-plurarl": "cleaning powder"
```
Comment (Night-Pryanik) [inline data/json/items/comestibles/med.json]: Typo.

**Comment by Night-Pryanik** [comment]
Looking good.

File: data/json/items/comestibles/med.json (hunk 2 above)
Comment (Broken-Admin) [reply data/json/items/comestibles/med.json]: I literally just fixed that in my last commit.

**Comment by Broken-Admin** [comment]
I'm glad

**Comment by ituluwituluwzev** [comment]
Chemically, what is this?

**Comment by Broken-Admin** [comment]
Alcohol Powder, https://goo.gl/YVW8Gm.

**Comment by Broken-Admin** [comment]
But as we already know, alcohol is disinfectant.

**Comment by ituluwituluwzev** [comment]
Ok...

**Comment by Broken-Admin** [comment]
@kevingranade Could you please review this on here since you blocked me on discord?

**Comment by kevingranade** [comment]
There is no indication in the link you provided: https://en.wikipedia.org/wiki/Alcohol_powder that this would be suitable for use as a disinfectant.  Do you have any sources that do indicate that?

**Comment by Broken-Admin** [comment]
https://goo.gl/8VGGQU, Use of alcohol ^ethanol as disinfectant.

**Comment by Broken-Admin** [comment]
If this is a bad topic I could just make it turn into an alcohol drink.

**Comment by kevingranade** [comment]
> Ethanol is typically used in concentrations of 70 percent

> According to Sato's web page, powdered alcohol contains 30.5% ethyl alcohol by volume in the state of powder.[10]

It's entirely too dilute to be useful.

**Comment by Broken-Admin** [comment]
Ight.

**Comment by Broken-Admin** [comment]
If I increased the amount of powder used in the crafting recipe could that settle out the concentration?

**Comment by kevingranade** [comment]
It doesn't matter what you do to the recipe, the actual substance has too
low a concentration of alchohol. You'd have to distill it to get the
concentration up, which defeats the purpose of "instant alchohol".
